src/quest_manager/views.py

```diff
@@ -51,6 +51,18 @@ class CategoryList(NonPublicOnlyViewMixin, LoginRequiredMixin, ListView):
     model = Category
 
 
+@method_decorator(staff_member_required, name='dispatch')
+class CategoryDetail(NonPublicOnlyViewMixin, DetailView):
+    model = Category
+
+    def get_context_data(self, **kwargs):
+
+        kwargs['heading'] = self.object.title
+        kwargs['c'] = self.object
```

Comment:
Is the c variable used anywhere?  I think you are using the `object` that is passed by default to the template now, correct? So you should be able to remove this.  And you can also remove ` kwargs['heading']` and use {{object.title}} directly in the template, so you can remove the ` get_context_data` method completely, which simplifies things a bit.

Reply:
Actually I don't think you are using heading anymore already, so this can just be removed.

Reply:
Removed the `get_context_data` method.